### ISSUE TITLE
Version issue every time

### ISSUE THREAD

**Issue by Ankit0080** [jamesblasco/flutter_preview#20]
And because flutter_cache_manager >=3.0.0-nullsafety.0 depends on http ^0.13.0 and preview >=0.0.5+2 depends on http ^0.12.0+2, cached_network_image ^3.1.0 is incompatible with preview >=0.0.5+2.
So, because Vliv_B2B depends on both cached_network_image ^3.1.0 and preview ^0.0.8, version solving failed.
pub get failed (1; So, because Vliv_B2B depends on both cached_network_image ^3.1.0 and preview ^0.0.8, version solving failed.)
exit code 1

**Comment by Rexios80** [comment]
Yeah this package is quite out of date. It doesn't even support null safety yet. I might work on it if I have time, but I'm unsure if I will be able to get it working.
